runtime/compiler/env/j9methodServer.cpp

```diff
@@ -2570,7 +2570,6 @@ TR_ResolvedRelocatableJ9JITServerMethod::staticAttributes(TR::Compilation * comp
 
    if (!resolveField)
```

Comment:
It's possible that `resolvedField` is `false` and `fieldInfoCanBeUsed` is `true`. Under these circumstances we would have written a NULL into `*address` which no longer happens in the new code.

Reply:
If `resolvedInfo` is false, it looked like we set `fieldInfoCanBeUsed` to false inside the `if (!resolveField)` block, so we'd end up execute the `if (!fieldInfoCanBeUsed)` block anyway. I could be misreading that.

Reply:
You are right.